QRCoder/ASCIIQRCode.cs

```diff
@@ -55,9 +54,34 @@ public string[] GetLineByLineGraphic(int repeatPerModule, string darkColorString
         var adjustmentValueForNumberOfCharacters = darkColorString.Length / 2 != 1 ? darkColorString.Length / 2 : 0;
         var verticalNumberOfRepeats = repeatPerModule + adjustmentValueForNumberOfCharacters;
         var sideLength = (QrCodeData.ModuleMatrix.Count - quietZonesModifier) * verticalNumberOfRepeats;
+        var qrCode = new string[sideLength];
+        var lineCapacity = (QrCodeData.ModuleMatrix.Count - quietZonesModifier) * repeatPerModule * darkColorString.Length;
+#if HAS_SPAN
+        if (darkColorString.Length == whiteSpaceString.Length && lineCapacity < 510)
+        {
+            Span<char> row = stackalloc char[lineCapacity];
```

Comment:
This gives a tad better perf, as stack allocating to a power of two is simpler for the JIT (due to zeroing, etc.).
```suggestion
            Span<char> row = stackalloc char[512].Slice(0, lineCapacity);
```